File: src/test/java/be/cegeka/orderit/service/eventstore/api/ResultAutoMatchedEventTest.java
package be.cegeka.orderit.service.eventstore.api;

public class ResultAutoMatchedEventTest extends BaseInternalAnalysisEventTest {

    @Override
    protected AbstractInternalAnalysisEvent item() {
        return new ResultAutoMatchedEvent(new InternalAnalysisAggregateId("sdfg"), AnalysisNumber.from(95067), "+", "unit", "device", "obsGroupId", "user");
    }
}

File: src/test/kotlin/be/cegeka/orderit/service/eventstore/api/ResultAutoMatchedEventTest.kt
package be.cegeka.orderit.service.eventstore.api

import be.cegeka.orderit.service.eventstore.api.AnalysisNumber.Companion.from

class ResultAutoMatchedEventTest : BaseInternalAnalysisEventTest() {

    override fun item(): AbstractInternalAnalysisEvent {
        return ResultAutoMatchedEvent(InternalAnalysisAggregateId("sdfg"), from(95067), "+", "unit", "device", "obsGroupId", "user")
    }
}